termlink/gsea.py

```diff
@@ -67,16 +68,26 @@ def execute(args):
     Args:
         args:   command line arguments from argparse
     '''
+
     uri = urlparse(args.uri)
     if uri.scheme != 'file':
         raise ValueError(f"uri.scheme '{uri.scheme}' is not supported")
 
-    filename = path.basename(uri.path) 
-    if not match(_filename_regex, filename):
+    input_file = uri.path
+    if not match(_filename_regex, os.path.basename(input_file)):
         raise ValueError(
-            f"File type is incorrect. Expected to match regular expression: '{_filename_regex}'. Found '{filename}'.")
+            f"File type is incorrect. Expected to match regular expression: '{_filename_regex}'. Found '{input_file}'.")
 
     schema = RelationshipSchema()
-    for relationship in _get_relationships(uri):
-        o = json.dumps(schema.dump(relationship)) 
-        print(o)
+
+    if args.output:
+        open('file.txt', 'w').close()
```

Comment:
?

Reply:
This is just a trick to clear out the file if it exists on the drive since we append to it below, not really needed for task service since the dir should be empty by default but is handy when running this locally.  I can drop it if you would rather local runs be cleanup by hand

Reply:
It's opening `file.txt`, not `input_file`.

Reply:
doh!, yeah, sorry, it should be `args.output` that's is cleared, but I'll just remove it since keeping it in for local runs does feel a bit sloppy.